index.bs

```diff
@@ -800,6 +800,8 @@ To <dfn>serialize an [=attribution report=]</dfn> |report|, run the following st
 
 1. Let |data| be a [=map=] of the following key/value pairs:
 
+    : "`attribution_destination`"
+    :: |report|'s [=attribution report/attribution destination=]
```

Comment:
We need to pass this in as a string given the requirements of https://infra.spec.whatwg.org/#convert-an-infra-value-to-a-json-compatible-javascript-value